backend/queries/events.js

```diff
@@ -27,6 +27,7 @@ const getAllEvents = async (vName, topic, instructor, upcoming, past) => {
     cohorts.cohort,
     cohorts.cohort_id,
     materials_url,
+    event_declaration,
```

Comment:
It's `event_duration` instead of `event_declaration`, can you please rename it into all this file?

Reply:
I was also expecting to see an `ARRAY_AGG` for confirmed volunteers since you used it in you last pull request `event.v_email.forEach(email => vEmails += `&add=${email}`)` to have the confirmed volunteers emails adding to the google calendar link.
We can event do it here or manage it in the front end as well, we will talk about that later on

Reply:
I renamed them in the query